controls/apps.py

```diff
@@ -1,5 +1,44 @@
 from django.apps import AppConfig
 
+MODULE_NAME = "controls"
+
+DEFAULT_CFG = {
+    "gql_mutation_create_families_perms": ['101002'],
+    "gql_mutation_update_families_perms": ['101003'],
+    "gql_mutation_create_insurees_perms": ["101102"],
+    "gql_mutation_update_insurees_perms": ["101103"],
+    "gql_mutation_create_policies_perms": ['101202'],
+    "gql_mutation_edit_policies_perms": ['101203'],
+    "gql_mutation_renew_policies_perms": ["101205"],
+    "gql_mutation_create_premiums_perms": ["101302"],
+    "gql_mutation_update_premiums_perms": ["101303"],
```

Comment:
These are already defined in the specific modules and should not be defined/configured here. You should take the rights from the specific modules as they might be configured differently. Single point of truth.

Reply:
Fixed